htdocs/core/lib/functions2.lib.php

```diff
@@ -814,7 +814,7 @@ function get_next_value($db,$mask,$table,$field,$where='',$objsoc='',$date='',$m
     //print "masktri=".$masktri." maskcounter=".$maskcounter." maskraz=".$maskraz." maskoffset=".$maskoffset."<br>\n";
 
     // Define $sqlstring
-    $posnumstart=strrpos($maskwithnocode,$maskcounter);	// Pos of counter in final string (from 0 to ...)
+    $posnumstart=mb_strrpos($maskwithnocode,$maskcounter, 'UTF-8');	// Pos of counter in final string (from 0 to ...)
```

Comment:
Yes I did the test and run without problem.
The code you suggest a better solution.